lib/kernel-manager.coffee

```diff
@@ -11,6 +11,7 @@ module.exports =
 class KernelManager
     constructor: ->
         @_runningKernels = {}
+        @_kernelSpecs = []
```

Comment:
I think `@_kernelSpecs` should be initialised with the user settings; i.e. `@_kernelSpecs = @parseKernelSpecSettings()`. Is there a case when we wouldn't this to happen?